### DISCUSSION TITLE
Welcome to wire discussions!

### DISCUSSION THREAD

**Discussion by Anticept** [wiremod/wire#2114] [General]
## 👋 Welcome!
  We’re evaluating discussions as a replacement for Reddit. Feel free to check things out!

  Please remember the code of conduct. Enjoy!

**Comment by MTNdew2** [comment]
Advanced duplicator 2 dupes and Expression 2 files are where they are supposed to be, but no longer show in game. Not even recopying the files to their respective directories works. How do I fix this?

**Reply by Anticept** [reply]
Clear cache and check again

**Comment by vurvdev** [comment]
Discussions are replacing suggestion issues. Please only post suggestions here now, we'll just convert any issues to here anyway.
